pages/settings/index.html

```diff
@@ -24,68 +24,72 @@
 </head>
 
 <body>
-  <section id="myProfile">
-    <h1>My Profile</h1>
-    <p>Update your photo and personal details here.</p>
-    <form id="profileForm">
-      <label for="name">Name</label>
-      <input type="text" id="name" />
-      <label for="email">Email address</label>
-      <input type="email" id="email" />
-    </form>
-    <h2>Password</h2>
-    <button id="changePwBtn">Change Password</button>
-    <h2>Your photo</h2>
-    <p>This will be displayed on your profile.</p>
-    <img src="#" alt="#" id="profilePhoto" />
-
-    <div id="fileDropWrapper">
-      <p>Drop your image here, or click to browse</p>
-    </div>
-
-    <button type="submit" id="saveProfileInfoBtn" form="profileForm">
-      Save
-    </button>
-  </section>
-  <section id="settings">
-    <h1>Settings</h1>
-    <p>Update your preferences here</p>
-
-    <div class="form-check form-switch">
-      <img src="#" alt="#" />
-      <!-- icon -->
-      <label class="form-check-label" for="pushNotificationSwitch">Push Notifications</label>
-      <input class="form-check-input" type="checkbox" role="switch" id="pushNotificationSwitch" checked />
-    </div>
-    <div id="changePwPopup">
-      <form id="changePwForm">
-        <label for="oldPw">Old Password</label>
-        <input type="password" id="oldPw" />
-        <label for="newPw">New Password</label>
-        <input type="password" id="newPw" />
-        <label for="newPwConfirm">Confirm New Password</label>
-        <input type="password" id="newPwConfirm" />
-        <button id="changePwSaveBtn">Save & Close</button>
-      </form>
-    </div>
-  </section>
-  <section id="deleteAccount">
-    <h1>Delete Account</h1>
-    <p>
-      If you no longer want to use CampBuddy, you can permanently delete your
-      account. You can’t undo this action!
-    </p>
-    <button id="deleteAccountBtn">Delete Account</button>
-    <div id="deleteAccountConfirm">
-      <h2>Are you sure you want to delete your account?</h2>
-      <button id="deleteAccountYesBtn">Yes</button>
-      <button id="deleteAccountNoBtn">No</button>
-    </div>
-  </section>
+	<section id="myProfile">
+		<h1>My Profile</h1>
+		<p>Update your photo and personal details here.</p>
+
+		<form id="profileForm">
+			<label for="name">Name</label>
+			<input type="text" id="name" required />
+			<label for="email">Email address</label>
+			<input type="email" id="email" readonly />
+		</form>
+
+		<h2>Password</h2>
+		<button id="resetPwBtn">Reset Password</button>
+
+		<div id="resetPwPopup">
+			<button id="resetPwSaveBtn">Reset Password</button>
+			<button id="resetPwCanelBtn">Cancel</button>
+		</div>
+
+		<h2>Your photo</h2>
+		<p>This will be displayed on your profile.</p>
+		<img src="#" alt="User profile picture" id="profilePhoto" />
+
+		<div id="fileDropWrapper">
+			<label for="inputImage" id="dropArea">
+				<input type="file" accept="image/*" id="inputImage" hidden />
+				<p>Drop your image here, or click to browse</p>
```

Comment:
Maybe is better to use a <span> here instead of <p>

Reply:
fixed in [649335e](https://github.com/whamester/WMDD-4885-Project/pull/15/commits/649335e40952afeb41a8595865ac3de6a6f70c71)